src/components/SingleArticle.jsx

```diff
@@ -21,35 +22,46 @@ function SingleArticle() {
 				setError(err);
 				setIsLoading(false);
 			});
-	}, []);
+	}, [article]);
 
+	const handleClick = () => {
+		setUserVotes(userVotes + 1)
```

Comment:
Generally it's bad practice to refer to the state within the set state function. Can you think of a better way within this set state invocation to grab the current value of that state?